packages/indy-vdr/src/dids/IndyVdrIndyDidRegistrar.ts

```diff
@@ -222,41 +268,44 @@ export class IndyVdrIndyDidRegistrar implements DidRegistrar {
   private async registerPublicDid(
     agentContext: AgentContext,
     pool: IndyVdrPool,
-    submitterDid: string,
-    submitterVerkey: string,
-    targetDid: string,
-    verkey: string,
+    unqualifiedSubmitterDid: string,
+    submitterSigningKey: Key,
+    unqualifiedDid: string,
+    signingKey: Key,
     alias?: string,
     role?: string,
     diddocContent?: Record<string, unknown>
   ) {
     try {
-      agentContext.config.logger.debug(`Register public did '${targetDid}' on ledger '${pool}'`)
+      agentContext.config.logger.debug(`Register public did '${unqualifiedDid}' on ledger '${pool}'`)
 
       // FIXME: Add diddocContent when supported by indy-vdr
       if (diddocContent) {
         throw new IndyVdrError('diddocContent is not yet supported')
       }
 
-      const request = new NymRequest({ submitterDid, dest: targetDid, verkey, alias })
-
-      const signingKey = Key.fromPublicKeyBase58(submitterVerkey, KeyType.Ed25519)
+      const request = new NymRequest({
+        submitterDid: unqualifiedSubmitterDid,
+        dest: unqualifiedDid,
+        verkey: signingKey.publicKeyBase58,
+        alias,
+      })
 
-      const response = await pool.submitWriteRequest(agentContext, request, signingKey)
+      const response = await pool.submitWriteRequest(agentContext, request, submitterSigningKey)
 
-      agentContext.config.logger.debug(`Registered public did '${targetDid}' on ledger '${pool.indyNamespace}'`, {
+      agentContext.config.logger.debug(`Registered public did '${unqualifiedDid}' on ledger '${pool.indyNamespace}'`, {
```

Comment:
Do you think it could be convenient to log using the full did format? Aligned with the way we are logging ledger objects in AnonCredsRegistry

Reply:
Yeah that would be convenient, but it means we need to pass either both the qualified and unqualified did or parse it again (which as overhead). 

What's your preference?

Reply:
Oh I see.. I think it's OK to use the unqualified in that case. Maybe not calling it a 'public did' but simply id or namespace id. But again, it's just some nitpicking here.